en/advanced/gimbal_control.md

```diff
@@ -51,13 +51,12 @@ For example, if the `TELEM2` port on the flight controller is unused you can con
 PX4 can automatically create a gimbal manager for a connected PWM gimbal or the first MAVLink gimbal device with the same system id it detects on any interface.
 It does not automatically create gimbal manager for any other MAVLink gimbal devices that it detects.
 
-You can support additional gimbals provided that they:
+You can support additional MAVLink gimbals provided that they:
 
-- implement the gimbal _manager_ protocol
+- Implement the gimbal _manager_ protocol.
 - Are visible to the ground station and PX4 on the MAVLink network.
   This may require that traffic forwarding be configured between PX4, the GCS, and the gimbal.
-- Each gimbal must have a unique component id.
-  For a PWM connected gimbal this will be the component ID of the autopilot
```

Comment:
FYI @julianoes - this is not relevant in a discussion about the additional gimbals. Note that we should require that the device ID for the autopilot connected camera is set to 1 by default, and reject other values.

Reply:
In theory that's there:
https://github.com/PX4/PX4-Autopilot/blob/fd87cd682db5360de4408aefa1b93bad1eaa1d3d/src/modules/gimbal/output_rc.cpp#L65-L66

Reply:
In practice it seems to be correct for GIMBAL_MANAGER_STATUS:

![image](https://github.com/user-attachments/assets/c9f92144-b503-4744-b7b8-6c8903855378)

But wrong for GIMBAL_DEVICE_ATTITUDE_STATUS:

![image](https://github.com/user-attachments/assets/ec0aaab1-c988-4bfa-862b-ce027eb4bd2e)

And here is a fix: https://github.com/PX4/PX4-Autopilot/pull/24123

Reply:
> In theory that's there: https://github.com/PX4/PX4-Autopilot/blob/fd87cd682db5360de4408aefa1b93bad1eaa1d3d/src/modules/gimbal/output_rc.cpp#L65-L66

That defines it in `OutputRC` - I assume that is the oddly named equivalent of the `AUX` (PWM) gimbal? Cool!

We probably don't yet report errors on non-1 values in the mission though, do we? [don't bother looking if you don't know :-)]

Reply:
Thanks for this. Good to get it clear.